.github/workflows/prod-build.yml

```diff
@@ -310,36 +317,53 @@ jobs:
           # Info about which CONTENT_* environment variables were set and to what.
           echo "CONTENT_ROOT=$CONTENT_ROOT"
           echo "CONTENT_TRANSLATED_ROOT=$CONTENT_TRANSLATED_ROOT"
-          yarn build:sw
-          yarn build:client
-          yarn build:ssr
-
-          cp assets/prod/robots.txt client/build/robots.txt
 
-          yarn rari content sync-translated-content
-          yarn rari git-history
+          mkdir -p out/static
 
-          yarn rari build --all --issues client/build/issues.json --templ-stats
+          npm run rari content sync-translated-content
+          npm run rari git-history
 
-          du -sh client/build
+          npm run rari build -- --all --issues out/issues.json --templ-stats
 
           # SSR all pages
-          yarn render:html
+          npm run build
```

Comment:
Nit: Does `npm run build` always build into `out/` or does it recognize `$BUILD_OUT_ROOT`? (If it doesn't, would be good to document.)